minimal-cases/issue-2451/Makefile

```diff
@@ -0,0 +1,10 @@
+# For a long term test build stdlib-debug or stdlib
+# using the standard Makefile and then run test-long-term
+# to ./stdlib --sequential in a loop with logs captured
+# in stdlib-ymd-hms.ns.txt.
+test-long-term:
+	while true; do echo "Date: `date +%y%m%d-%H%M%S.%N`"; ./stdlib --sequential; echo "stdlib done"; done 2>&1 | tee stdlib-`date +%y%m%d-%H%M%S.%N`.txt
```

Comment:
I don't think this target will work in isolation - you should add the bit that builds the `stdlib` binary to this small Makefile.